src/renderer/components/wallet/Settings.tsx

```diff
@@ -42,44 +42,20 @@ const UserAccounts: UserAccountType[] = [
         name: 'Main',
         address: 'tbnb1vxutrxadm0utajduxfr6wd9kqfalv0dg2wnx5y',
```

Comment:
Pls show address of Binance Account (via `address$` of `services/binance`)). I've forgotten to mention it in the descriptioin of #344)